managed/services/agents/encryption.key

```diff
@@ -0,0 +1 @@
+CKSprv8IEmQKWAowdHlwZS5nb29nbGVhcGlzLmNvbS9nb29nbGUuY3J5cHRvLnRpbmsuQWVzR2NtS2V5EiIaIMxMAgxuYisbREYQUSmC4DsGOHJ/EVBhQa5SC0+j9cj9GAEQARikqa7/CCAB
```

Comment:
Not sure, but it shouldn't be committed, right? Or there is reason to do that?

Reply:
oh. good point. need to add it to gitignore.